pkg/xds/v2/adssubscriber.go

```diff
@@ -84,7 +85,13 @@ func (adsClient *ADSClient) receiveThread() {
 			}
 			resp, err := sc.Recv()
 			if err != nil {
-				log.DefaultLogger.Infof("[xds] [ads client] get resp timeout: %v, retry after 1s", err)
+				// rpc error: code = Unavailable desc = transport is closing
+				if strings.Contains(err.Error(), "transport is closing") {
+					adsClient.reconnect()
```

Comment:
maybe conflict with `adsClient.reconnect()` in `sendThread()`.

Reply:
remove it.